src/Form/IslandoraDateFilter.php

```diff
@@ -166,7 +166,7 @@ public function buildForm(array $form, FormStateInterface $form_state, array $el
       '#name' => 'date_filter_facet_field_' . $form_key,
       '#form_key' => $form_key,
     ];
-    $form['#attached']['library'][] = 'core/jquery.ui.datepicker';
+    $form['#attached']['library'][] = 'islandora_solr/islandora_solr_facets.datepicker.js';
```

Comment:
This is not a lib? `islandora_solr/datepicer` maybe? If this is so, it questions how the related work was tested.

Reply:
I'm dumb, I was looking for the date slider and not the datepicker during testing of console logs